### ISSUE TITLE
Uncaught TypeError: n.map is not a function

### ISSUE THREAD

**Issue by slatifi** [femioladeji/react-slideshow#126]
When trying to use this package, I got the following error in the Chrome console: Uncaught TypeError: n.map is not a function.

Below is a screenshot of the code I was using:

![image](https://user-images.githubusercontent.com/43909123/113148591-e9472380-9229-11eb-94b7-c2db84dedbb9.png)

I have the following dependencies related to this project:

"react-slideshow-image": "^3.4.3"
"react": "^17.0.2"
"react-dom": "^17.0.2"
"css-loader": "^5.2.0"
"style-loader": "^2.0.0"

Any help would be much appreciated

**Comment by slatifi** [comment]
Ignore the underlined Slide import - this is just an editor feature.

**Comment by femioladeji** [comment]
@slatifi you're seeing the error because you only have one direct child. A quick fix will be something like this
```js
const Slideshow = () => {
    return (
      <div>
        <Slide easing="ease">
          {[1].map(() => <div className="each-slide">
            <div style={{'backgroundImage': `url(${slideImages[0]})`}}>
              <span>Slide 1</span>
            </div>
          </div>)}
        </Slide>
      </div>
    )
};
```

I'll release a new version soon that will effectively solve this

**Comment by femioladeji** [comment]
@slatifi I've released a new version `v3.4.5` you can re-install the package
